yarn.lock

```diff
@@ -6298,7 +6305,7 @@ config-chain@^1.1.11:
     ini "^1.3.4"
     proto-list "~1.2.1"
 
-configstore@4.0.0:
+configstore@4.0.0, configstore@^4.0.0:
```

Comment:
while you're at it could you update https://github.com/gatsbyjs/gatsby/blob/f5a85af786e2be73afcaf94f8da68f1e08bb42f9/packages/gatsby-telemetry/package.json#L14 to have a ^ `"configstore": "^4.0.0"`